scripts/scrubber.py

```diff
@@ -24,7 +24,7 @@ def parents_are_newest(self):
             print(np.difference(n))
 
     def version_number_check(self):
-        x = self.dbu.session.execute("SELECT max(interface_version), max(quality_version), max(revision_version) FROM file").fetchone()
+        x = self.dbu.session.execute().fetchone()
```

Comment:
This doesn't fix the issue, it just doesn't do what we're trying to do. You need the explicit cast like you did in 83c49641995daf35a6cbb0b3683dd7e0251d10f6